httpdConf/stagingHttpd.conf

```diff
@@ -0,0 +1,409 @@
+#
+# This is the main Apache HTTP server configuration file.  It contains the
```

Comment:
It won't let me comment where I actually want to comment. Renaming the httpd.conf to prod httpd.conf means the prod simlink needs to updated. I think this should be left as is and changed when we fix prod.